src/components/backLink/test.js

```diff
@@ -5,8 +5,9 @@ import ReactDOM from "react-dom";
 import BackLink from "./";
 
 describe("Back Link", () => {
+  const testFn = () => false;
```

Comment:
this test should use sinon to confirm that the click handler is called when the button is clicked, but happy for this to go in as a TODO: comment for now